spec/models/book_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+describe Book do
+  it 'validates presence of genre' do
```

Comment:
Take care here, use the same syntax that's provided on the card.

```
  it do
    should validate_presence_of(:first_name)
  end
```
This syntax is provided by shoulda matchers, https://github.com/thoughtbot/shoulda-matchers

Reply:
Idem with every presence validation